_layouts/exit-when.html

```diff
@@ -0,0 +1,8 @@
+{% include header.html %}
```

Comment:
Do we need a separate layout for this?

Reply:
Hmm not sure; what's your opinion on that?

Reply:
I'll look again once it's merged.